src/kirby/components/list/list.component.spec.ts

```diff
@@ -128,14 +128,16 @@ describe('ListComponent', () => {
   });
 
   describe('function: onItemSelect', () => {
-    it('should emit the selected item', () => {
+    it('should emit the selected item and mark it as selected', () => {
       spyOn(component.itemSelect, 'emit');
-      const itemToBeSelected = { value: 'this is a dummy item' };
+      const itemToBeSelected = { value: 'this is a dummy item', selected: false };
+      component.items = [itemToBeSelected];
 
       component.onItemSelect(itemToBeSelected);
 
       expect(component.itemSelect.emit).toHaveBeenCalledTimes(1);
       expect(component.itemSelect.emit).toHaveBeenCalledWith(itemToBeSelected);
+      expect(itemToBeSelected.selected).toBe(true);
```

Comment:
Same as other comment regarding the use of properties on the item.

Reply:
Removed it